commit 696439f1ae107c6a6fe0bc4949b0ddb0605dc939 (refs/changes/26/1026/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-11-30T16:45:10-05:00 (2 years, 10 months ago)
    
    joyent/node-triton#149 resurrected example-get-account
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/examples/example-get-account.js b/examples/example-get-account.js
index 043f5d5..8bcb3cf 100755
--- a/examples/example-get-account.js
+++ b/examples/example-get-account.js
@@ -12,20 +12,22 @@ var bunyan = require('bunyan');
 var cloudapi = require('../lib/cloudapi2');
 
 var log = bunyan.createLogger({
-    name: 'test-get-account',
+    name: 'example-get-account',
     level: 'trace'
 })
 
-var USER = process.env.SDC_USER || process.env.SDC_ACCOUNT || 'bob';
+var ACCOUNT = process.env.SDC_ACCOUNT || 'bob';
+var USER = process.env.SDC_USER;
 var KEY_ID = process.env.SDC_KEY_ID || 'b4:f0:b4:6c:18:3b:44:63:b4:4e:58:22:74:43:d4:bc';
 
 var sign = auth.cliSigner({
     keyId: KEY_ID,
-    user: USER,
+    user: ACCOUNT,
     log: log
 });
 var client = cloudapi.createClient({
     url: 'https://us-sw-1.api.joyent.com',
+    account: ACCOUNT,
     user: USER,
     version: '*',
     sign: sign,
